### DISCUSSION TITLE
is there a way to transfer favourites and playlists from Vi music?

### DISCUSSION THREAD

**Discussion by ChickensioDev** [fast4x/RiMusic#399] [Q&A] [answered]
hi, im an avid user of vi music, but i was sad to see that it was dead with no updates, then i saw the forks and saw this. i was wondering if there was a way to transfer all my favourites and playlists from vi music to this app somehow?

**Comment by Enigma1309** [comment, marked as answer]
Yes it is but it's kind of a trick

Also there is a download feature in the app, just before exporting try adding all the offline songs to favorite/other playlists (you can follow trick 3 from below link)

https://github.com/25huizengek1/ViMusic/discussions/38#discussioncomment-7845575

**Reply by ChickensioDev** [reply]
ok i just found the way, i just made a backup on vi music and used restore feature from ri music

**Reply by Enigma1309** [reply]
glad it worked 😀

**Reply by fast4x** [reply]
Added to faqs on wiki

**Comment by fast4x** [comment]
RiMusic is fully compatible with ViMusic backup
